casual.html

```diff
@@ -59,18 +58,33 @@
 </div>
 <!-- end topbar-->
 <div class="contentContainer" id="contentContainer">
-<div class="sidebar" id="sidebar">
-	<div class="screenHeight">
-	<div id="sidebarContent" class="sidebarContent resizableWidthContainer ">
+<div class="guideFrame">
+<article id="guide" class="mainPanel">
+	<div class="info">The Casual Guide will be available soon! In the meantime, the <a href="https://prclive.run/wiki/Main_Page">wiki</a> can help you learn about specific objectives in the run.</div>
+
+	<!-- BEGIN FOOTER-->
+<footer>
+<hr/>
+<p>Licensed under GPLv3.
+ <a href="https://github.com/MichaelEbert/OblivionProgressTracker">Contribute</a>
+ PRCLive twitch
+ </p>
+</footer>
+<!-- END FOOTER-->
+</article>
+
+<div class="resizer" id="dragMe"></div>
+
+<div class="sidePanel" id="sidebar">
+	<div id="sidebarContent" class="sidebarContent">
 		<div id="sidebarLinks">
-			<a href="./help.html" target="_blank">🛈</a>
-			<a href="./map.html?topbar=false" target="_blank" title="Show Map">🗺️</a>
+			<a href="./help.html" target="myframe" title="Help Page">🛈</a>
```

Comment:
The way I had it set up before, all links that should open in the sidebar were default target="_blank", so they work without js, and then if the sidebar is enabled and visible, we set all the _blank links to "myframe".

Reply:
though i guess technically it doensn't matter here as those links are only visible during sidebar...

Reply:
Yeah the fact that the previous implementation took every link from every part of the page to update it seemed potentially problematic to me so I opted for only manipulating link targets of the actual guide content. In this configuration we can actually have the option to change settings on the actual guide page and see them update properly (as far as I can tell) if we ever wanted to have a "quick settings" popout or something.